.changeset/olive-toes-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roll-network/api': minor
+---
+
+Change response formatting in Get Platform User Balance so that it displays the token symbol correctly
```

Comment:
The examples are not part of NPM, so we have to write the changelog regarding the changes in the package itself :)

```suggestion
Updated getPlatformUserBalance response type
```

Reply:
Thanks for pointing it out @teimurjan, I added the changelog earlier as I was making changes in the packages/api. No need for it now.